src/core_atmosphere/physics/mpas_atmphys_driver.F

```diff
@@ -260,23 +268,23 @@ subroutine physics_driver(domain,itimestep,xtime_s)
 !$OMP PARALLEL DO
        do thread=1,nThreads
           call driver_sfclayer(itimestep,block%configs,mesh,diag_physics,sfc_input, &
-                               cellSolveThreadStart(thread), cellSolveThreadEnd(thread))
+                               cellSolveThreadStart(thread),cellSolveThreadEnd(thread))
        end do
 !$OMP END PARALLEL DO
        call deallocate_sfclayer(block%configs)
     endif
 
     !call to 1d ocean mixed-layer model
-    if(config_oml1d) call driver_oml1d(block%configs, mesh, diag, diag_physics, sfc_input)
+    if(config_oml1d) call driver_oml1d(block%configs,mesh,diag,diag_physics,sfc_input)
 
     !call to land-surface scheme:
     if(config_lsm_scheme .ne. 'off') then
```

Comment:
@tanyasmirnova this is the if statement you said need to change

Reply:
The line 281 should be:
if(config_lsm_scheme .eq. 'sf_noah' or. config_lsm_scheme .eq. 'sf_ruc') then